src/views/directMessages/components/messages.js

```diff
@@ -62,12 +61,24 @@ class MessagesWithData extends React.Component<Props, State> {
     }
     // force scroll to bottom when a message is sent in the same thread
     if (prev.data.messages !== data.messages && contextualScrollToBottom) {
+      setTimeout(() => this.props.forceScrollToBottom());
```

Comment:
I removed this line and things still work - any thing you found on your end that this is needed for?